sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -1090,15 +1102,26 @@ private void logStartupInfo(StopWatch stopwatch, CosmosAsyncClient client) {
 
         if (logger.isInfoEnabled()) {
             long time = stopwatch.getTime();
+            String diagnosticsCfg = "";
+            String tracingCfg = "";
+            if (client.getClientTelemetryConfig() != null) {
+                diagnosticsCfg = client.getClientTelemetryConfig().toString();
+            }
+
+            DiagnosticsProvider provider = client.getDiagnosticsProvider();
+            if (provider != null) {
+                tracingCfg = provider.isEnabled() + ", " + provider.isRealTracer();
+            }
+
             // NOTE: if changing the logging below - do not log any confidential info like master key credentials etc.
             logger.info("Cosmos Client with (Correlation) ID [{}] started up in [{}] ms with the following " +
                     "configuration: serviceEndpoint [{}], preferredRegions [{}], connectionPolicy [{}], " +
                     "consistencyLevel [{}], contentResponseOnWriteEnabled [{}], sessionCapturingOverride [{}], " +
-                    "connectionSharingAcrossClients [{}], clientTelemetryEnabled [{}], proactiveContainerInit [{}].",
+                    "connectionSharingAcrossClients [{}], clientTelemetryEnabled [{}], proactiveContainerInit [{}], diagnostics [{}], tracing [{}]",
```

Comment:
do we need a change log for this change?

Reply:
No - intentionally makred it as [Internal] - which in .Net we use as sign to not include in change log - content of a log I think just causes noise in changelog